lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
```

Comment:
I actually think that since we added schema to rules, we should create an eslint-plugin just for rules and consume it. We could check for correct schema and documentation with it.